sentry_sdk/integrations/aws_lambda.py

```diff
@@ -25,6 +27,49 @@
 
     F = TypeVar("F", bound=Callable[..., Any])
 
+# Constants
+TIMEOUT_THRESHOLD_MILLIS = 1500  # Minimum time required to capture TimeoutError
+SECONDS_CONVERSION_FACTOR = 1000.0
+
+
+def _wrap_init_error(init_error):
+    # type: (F) -> F
+    def sentry_init_error(*args, **kwargs):
+        # type: (*Any, **Any) -> Any
+
+        # Fetch Initialization error details from arguments
+        error = json.loads(args[1])
```

Comment:
Can we move this down into line 47, no reason to do this work and potentially crash even if the integration is disabled

Reply:
Sure, I'll make this change.